wagtail/tests/test_streamfield.py

```diff
@@ -237,6 +237,22 @@ def test_can_append(self):
         self.assertEqual(fetched_body[1].block_type, "text")
         self.assertEqual(fetched_body[1].value, "bar")
 
+    def test_can_append_on_queried_instance(self):
+        # The test is analog to test_can_append(), but instead of working with the
+        # in-memory version from JSONStreamModel.objects.create(), we query a fresh
+        # instance from the db.
```

Comment:
Maybe I should also write about the crash or the issue that has been fixed, otherwise the reasoning behind this small change compared to the original test may be unclear to a new reader?

Reply:
tried it hiere: https://github.com/wagtail/wagtail/pull/12322/files#diff-eda527fd71aaf539c4e4c6ac1157cfbf7e02ba9bb77eb6ab439b398c57ffb4aaR244-R245